product_docs/docs/biganimal/release/getting_started/identity_provider/auth0.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "Using Auth0 as your identity provider"
+navTitle: "Auth0"
+description: "Using Auth0 as your identity provider for your Azure or AWS account"
+---
+
+## Prerequisites
+
+- To connect BigAnimal to Auth0, you must be assigned the Auth0 admin role.
+
+- A unique URL and access code is provided in an email from cloudcare@enterprisedb.com. Contact [cloudcare@enterprisedb.com](mailto:cloudcare@enterprisedb.com) if you don't receive the email. The URL becomes invalid after you set up your identity provider with BigAnimal. If you have issues with the code or identity provider setup, contact [Support](/biganimal/release/overview/support).
+
+## Set up BigAnimal with Auth0
+
+1. Open the link in the email sent from cloudcare@enterprisedb.com to access the Set Up Identity Provider page in BigAnimal.
+
+1. You will be copying and pasting between BigAnimal and Auth0 so in a separate browser tab or window, log into [auth0.com](https://auth0.com/) and navigate to the Applications page and select **Create Application**.
+
+1. In the Create application dialog, enter a name and choose **Native** as the application type. Select **Create**.
+1. On the Connections tab or the page for your application, enable the toggles for the databases containing your user names (Username-Password-Authentication, GitHub, or Google/Gmail).
+1. Select the Settings tab of the page for your application, scroll down, and expand the **Advanced Settings** section.
+1. Select the Certificates tab in Advanced Settings and download the certificate in PEM format.
+1. Select the Addon tab in Advanced Settings and enable the **SAML2 Web App** toggle. 
```

Comment:
The `Addons` tab is not in the Advanced Settings, it's one tab belongs to the app. Maybe something like `Select the Addons tab in the page for your application and ...`